1_model_training/docker/Dockerfile

```diff
@@ -21,7 +14,7 @@ RUN apt-get update && apt-get install -y \
     libgl1-mesa-dev \
     wget
 
-# Copy this version of of the model garden into the image
+# Copy this version of the model garden into the image
 COPY models/research/object_detection /home/tensorflow/models/research/object_detection 
 
```

Comment:
@ruddyscent Kindly, it seems there is an attribute error that appears in logs which does not affect training completion or trained model's deployment. By making a quick research, we can find this error is related to the version of `pillow` which could be resolved by using:


`!pip install pillow==9.5`
or
`!pip install pillow --upgrade`

However, I think it does not work by installing manually, so do you have any workarounds here, please?

![image](https://github.com/udacity/cd2688-object-detection-in-urban-environment-project/assets/31706646/4f7db737-c9c2-47fd-a660-0717ec9937a2)

Reply:
We need to change the pillow version of the image that runs in SageMaker, not the environment that runs 1_train_model.ipynb. I've tested with the version you provided, and it doesn't throw the error message. So, I'll commit the fix now.